### PR TITLE
Move requirements to above usage.

### PR DESCRIPTION
**Pull request by n3wscott**
Usage breaks if the requirements are not met.

### CODE REVIEW

File: installer/README.md [hunk 1]
```diff
@@ -28,6 +28,31 @@ go get github.com/GoogleCloudPlatform/k8s-service-catalog/installer/cmd/sc
 
 After running the above command, `sc` should get installed in your GOPATH/bin dir.
 
+## Requirements
```
Comment (droot) [inline installer/README.md]: we should move it before installation I guess.

Comment (droot) [reply installer/README.md]: Pl. change the order of the links at the top as well.

Comment (n3wscott) [reply installer/README.md]: Done. Thanks!

**Comment by droot** [comment]
Thanks for doing it @n3wscott 👍

**Pull request merged**